store/migration/10001__init_schema.sql

```diff
@@ -859,8 +859,12 @@ CREATE TABLE repository (
     branch_filter TEXT NOT NULL DEFAULT '',
     -- Base working directory we are interested.
     base_directory TEXT NOT NULL DEFAULT '',
+    -- Base working directory to save sheet files.
```

Comment:
For new migration, we can no longer change the schema here and need to use up script. Please sync offline with @d-bytebase

Reply:
+1 I will provide an example this week.